app/models/rental.rb

```diff
@@ -140,10 +140,16 @@ def self.to_json_reservations
     arr
   end
 
-  def sum_amount
-    due = financial_transactions.where.not(transactable_type: Payment.name).sum(:amount)
-    paid = financial_transactions.where(transactable_type: Payment.name).sum(:amount)
-    due - paid # costs - payments
+  def sum_charges
```

Comment:
Nitpick, but `sum_charges` sounds like an imperative, where this method is more of a query than a command. I might go with `charge_sum`.